components/Footer.tsx

```diff
@@ -118,17 +140,42 @@ function Footer() {
             <ul className="font-medium text-gray-200">
               {footerServiceLinks.map((section, index) => (
                 <div key={index}>
-                  <h3 className="text-xl font-extrabold mt-4 mb-4 text-gray-200">{section.title}</h3>
+                  <h3 className="text-xl font-extrabold mt-5   mb-4 text-gray-200">
+                    {section.title}
+                  </h3>
                   <ul>
                     {section.child.map((link, linkIndex) => (
                       <li key={linkIndex} className="pt-2">
-                        <Link href={link.link} className="hover:text-primary " rel="noopener noreferrer" target="_blank" aria-label="footer service link">{link.name}</Link>
+                        <Link
+                          href={link.link}
+                          className="hover:text-primary "
+                          rel="noopener noreferrer"
+                          target="_blank"
+                          aria-label="footer service link"
+                        >
+                          {link.name}
+                        </Link>
                       </li>
                     ))}
                   </ul>
                 </div>
               ))}
             </ul>
+            <div className="mt-[25px]">
+              <div className="github-button p-2 bg-white rounded-lg max-w-[170px] mx-auto  ">
+                <a
+                  className="github"
+                  href="https://github.com/priyankarpal/ProjectsHut"
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  <img
```

Comment:
use `<Image>`

https://nextjs.org/docs/pages/api-reference/components/link

Reply:
```javascript
                  <Image
                    src="https://github.com/priyankarpal/ProjectsHut/assets/88102392/0fa9e8f0-349d-4cc8-8e01-1b6e2c27dbbb"
                    alt="Github-Share-Button"
                    width={144}
                    height={46}
                  />
                
```

This also Fixed